app/test_main.py

```diff
@@ -1,3 +1,34 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestConvertAnimalAgeToHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_age",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17])
```

Comment:
The test cases provided are correct and cover the examples given in the task description. However, the checklist suggests testing edge cases such as negative numbers, zero, or very large numbers. Consider adding these test cases to ensure the function handles all possible inputs correctly.